tests/php/SiteTreeSubsitesTest.php

```diff
@@ -47,37 +69,40 @@ public function testPagesInDifferentSubsitesCanShareURLSegment()
 
     public function testBasicSanity()
     {
-        $this->assertTrue(singleton('SiteTree')->getSiteConfig() instanceof SiteConfig);
+        $this->assertInstanceOf(SiteConfig::class, singleton(SiteTree::class)->getSiteConfig());
         // The following assert is breaking in Translatable.
-        $this->assertTrue(singleton('SiteTree')->getCMSFields() instanceof FieldList);
-        $this->assertTrue(singleton('SubsitesVirtualPage')->getCMSFields() instanceof FieldList);
-        $this->assertTrue(is_array(singleton('SiteTreeSubsites')->extraStatics()));
+        $this->assertInstanceOf(FieldList::class, singleton(SiteTree::class)->getCMSFields());
+        $this->assertInstanceOf(FieldList::class, singleton(SubsitesVirtualPage::class)->getCMSFields());
+        $this->assertTrue(is_array(singleton(SiteTreeSubsites::class)->extraStatics()));
     }
 
     public function testErrorPageLocations()
     {
-        $subsite1 = $this->objFromFixture('Subsite', 'domaintest1');
+        $this->markTestSkipped('needs refactoring');
+
+        $subsite1 = $this->objFromFixture(Subsite::class, 'domaintest1');
 
         Subsite::changeSubsite($subsite1->ID);
-        $path = SiteTreeSubsitesTest_ErrorPage::get_error_filename_spy(500);
+        $path = ErrorPage::get_filepath_for_errorcode(500);
 
-        $expected_path = 'error-500-'.$subsite1->domain().'.html';
+        $static_path = Config::inst()->get(ErrorPage::class, 'static_filepath');
+        $expected_path = $static_path . '/error-500-' . $subsite1->domain() . '.html';
         $this->assertEquals($expected_path, $path);
     }
 
     public function testCanEditSiteTree()
     {
-        $admin = $this->objFromFixture('Member', 'admin');
-        $subsite1member = $this->objFromFixture('Member', 'subsite1member');
-        $subsite2member = $this->objFromFixture('Member', 'subsite2member');
+        $admin = $this->objFromFixture(Member::class, 'admin');
+        $subsite1member = $this->objFromFixture(Member::class, 'subsite1member');
+        $subsite2member = $this->objFromFixture(Member::class, 'subsite2member');
         $mainpage = $this->objFromFixture('Page', 'home');
         $subsite1page = $this->objFromFixture('Page', 'subsite1_home');
         $subsite2page = $this->objFromFixture('Page', 'subsite2_home');
-        $subsite1 = $this->objFromFixture('Subsite', 'subsite1');
-        $subsite2 = $this->objFromFixture('Subsite', 'subsite2');
+        $subsite1 = $this->objFromFixture(Subsite::class, 'subsite1');
+        $subsite2 = $this->objFromFixture(Subsite::class, 'subsite2');
 
         // Cant pass member as arguments to canEdit() because of GroupSubsites
-        Session::set("loggedInAs", $admin->ID);
+        Session::set('loggedInAs', $admin->ID);
```

Comment:
`set` isn't static any more. If the purpose of this is to log the mocked user in then we should use `$this->logInAs($admin)`